frontend/client/types.ts

```diff
@@ -49,6 +49,14 @@ export type Service = {
   };
 };
 
+export type ServiceEnvVariable = {
+  name: string;
+  env_variable_name: string;
+  description: string;
+  value: string | number | boolean | null;
```

Comment:
Are these the only values allowed? Perhaps add another field 'type' ?

Reply:
Yes, but these types are only for type checking on frontend. Eventually, when they are exported as environment variables, there they only exist as strings.

Reply:
See comment below